src/server/upgrade/index.md

```diff
@@ -10,6 +10,64 @@ Make sure to always back up your database data before doing a migration.
 
 [[toc]]
 
+## Migration guide from 2024.2.x to 2025.2.x
+
+Get the latest <GitHubRepo id="MerginMaps/server/blob/master/docker-compose.yml" desc="docker-compose file" />  or update docker images manually to version `2025.2.2`.
+Perform the migration:
+
+<MigrationType type="CE" />
+::: tip Before you upgrade!
+Release 2025.2.2 brings significant changes on <MainPlatformName /> docker compose orchestration infrastructure.
+
+Previous individual `server` container is replaced by 3 service dedicated containers that split the core components of <MainPlatformName />, `server-gunicorn` the app, `celery-beat` Celery task scheduler and `celery-worker` a dedicated worker container for Celery tasks.
+:::
+
+1. Stop your running docker containers
+   ```bash
+       $ docker compose -f docker-compose.yml down # or similarly, based on your deployment
+       # INFO: After shutdown update the docker-compose.yml file to latest release
+   ```
+
+2. Double check if below environment variables are available and filled in `.prod.env` environment file. If not, add them.
```

Comment:
not sure, we want to duplicate these or make it common for both, EE an CE, I am happy either way

Reply:
I would leave it, since it's a 'step-by-step' approach under different scopes (EE and CE)